src/plugins/discover/public/application/main/discover_main_route.tsx

```diff
@@ -76,6 +77,8 @@ export function DiscoverMainRoute(props: Props) {
   const loadDefaultOrCurrentIndexPattern = useCallback(
     async (searchSource: ISearchSource) => {
       try {
+        debugger;
```

Comment:
I think you have forgotten to remove `debugger` here. It might be the reason of ci failures.